docs/style-guide.rst

```diff
@@ -1284,6 +1284,20 @@ Avoiding Naming Collisions
 This convention is suggested when the desired name collides with that of
 an existing state variable, function, built-in or otherwise reserved name.
 
+Underscore Prefix for Non-external Functions and Variables
+=====================================================
+
+* ``_singleLeadingUnderscore``
+
+This convention is suggested for non-external (``internal`` and ``private``) functions and non-external (``private`` or ``internal``) state variables (state variables without a visibility specified are default ``internal``).
+
+There are many reasons to consider this convention but we note just
+one here. When designing a smart contract the external surface area (functions
```

Comment:
```suggestion
When designing a smart contract,  the public facing API (functions
```
1. No need for the first line unless we're going to elaborate all of the reasons for this suggestion.
2. Not sure I've ever heard this being called `external surface area`